rabbitmq/src/main/java/io/micronaut/rabbitmq/intercept/RabbitMQIntroductionAdvice.java

```diff
@@ -404,6 +409,18 @@ private Optional<String> findRoutingKey(MethodInvocationContext<Object, Object>
                 .findFirst();
     }
 
+    private Optional<Boolean> findMandatoryFlag(MethodInvocationContext<Object, Object> method) {
+        Map<String, Object> argumentValues = method.getParameterValueMap();
+        return Arrays.stream(method.getArguments())
+                .filter(arg -> arg.getAnnotationMetadata().hasAnnotation(Mandatory.class))
+                .map(Argument::getName)
+                .map(argumentValues::get)
+                .filter(Objects::nonNull)
+                .map(Object::toString)
+                .map("true"::equalsIgnoreCase)
```

Comment:
use `StringUtils.TRUE` instead of `"true"`

Reply:
this code is a bit suboptimal from a performance perspective. Ideally you should use `method.getArguments()` and array iteration and then obtain the value from `getParameterValues()` array